platform/fabric/core/generic/rwset.go

```diff
@@ -56,6 +60,10 @@ func (c *RWSetLoader) GetRWSetFromEvn(txID string) (driver.RWSet, driver.Process
 }
 
 func (c *RWSetLoader) GetRWSetFromETx(txID string) (driver.RWSet, driver.ProcessTransaction, error) {
+	if !c.TransactionService.Exists(txID) {
```

Comment:
I would skip calling `Exists` because in the background it calls load, so we end up calling load twice.